### DISCUSSION TITLE
Technical meeting - 28/01/2022 - Action points

### DISCUSSION THREAD

**Discussion by FranckCo** [INTERSTAT/Statistics-Contextualized#27] [Meeting notes]
### Istat

- Put a nuts3_lau-it.csv file containing 2 columns : lau & nuts3 in the https://github.com/INTERSTAT/Statistics-Contextualized/tree/main/pilots/resources folder
- Delete https://github.com/INTERSTAT/Statistics-Contextualized/blob/main/LAU-NUTS3.IT.conversion and https://github.com/INTERSTAT/Statistics-Contextualized/blob/main/nuts3-it if not used
- Provide the OWL file for the ontology (mail to Franck)
- Specify the function, if any, mapping (pollutant, year) to URL of the XLSX file on the Ispra web site (additional question: is the structure of the Excel file the same for the different pollutant?)
- Specify target CSV format (before transformation to RDF) for the data and how to get there (beyond what Francesca already described in issue #17)

### Insee

- Send a mail on how to include diagrams in the GitHub repository
- Implement the SEP-AQ pipeline (at least Extraction and beginning of Transformation according to Istat specification) 

### Eng
- Francesca to contact Nicolas to set up a meeting to kick-start client app repository

### Fiware

- Fernando sends mail or completes https://github.com/INTERSTAT/Statistics-Contextualized/issues/10 to detail problems in the SDMX data

## Next week

Talk about issue #10 and follow up on previous points

**Comment by pafrance** [comment]
> ## Action points
> ### Istat
> * Put a nuts3_lau-it.csv file containing 2 columns : lau & nuts3 in the https://github.com/INTERSTAT/Statistics-Contextualized/tree/main/pilots/resources folder
> * Delete https://github.com/INTERSTAT/Statistics-Contextualized/blob/main/LAU-NUTS3.IT.conversion and https://github.com/INTERSTAT/Statistics-Contextualized/blob/main/nuts3-it if not used

Done. Waiting tor commit on the admin side

> * Provide the OWL file for the ontology (mail to Franck)

Done, <strike>no feedback from Franck</strike>

> * Specify the function, if any, mapping (pollutant, year) to URL of the XLSX file on the Ispra web site (additional question: is the structure of the Excel file the same for the different pollutant?)
> * Specify target CSV format (before transformation to RDF) for the data and how to get there (beyond what Francesca already described in issue [SEP data workflow: air quality data #17](https://github.com/INTERSTAT/Statistics-Contextualized/issues/17))

<strike>Did it, or so I thought but ... I don't find the post anymore. Was it deleted by mistake, I guess? Rewriting it now.</strike>
<strike>Compiling ISPRA dataset's column mappings</strike>
Done
